bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/wrappers/veneer/TestDataClientVeneerApi.java

```diff
@@ -204,53 +206,58 @@ public void testReadRows() throws IOException {
         .thenReturn(
             ImmutableList.of(Result.EMPTY_RESULT, EXPECTED_RESULT, EXPECTED_RESULT).iterator())
         .thenReturn(ImmutableList.<Result>of().iterator());
-    when(mockStreamingCallable.call(Mockito.eq(query), Mockito.any(GrpcCallContext.class)))
+    when(mockStreamingCallable.call(Mockito.any(Query.class), Mockito.any(GrpcCallContext.class)))
         .thenReturn(serverStream)
         .thenReturn(serverStream);
 
     ResultScanner resultScanner = dataClientWrapper.readRows(query);
     assertResult(Result.EMPTY_RESULT, resultScanner.next());
     assertResult(EXPECTED_RESULT, resultScanner.next());
 
-    doNothing().when(serverStream).cancel();
     resultScanner.close();
 
     ResultScanner noRowsResultScanner = dataClientWrapper.readRows(query);
     assertNull(noRowsResultScanner.next());
 
-    verify(serverStream).cancel();
     verify(mockDataClient, times(2)).readRowsCallable(Mockito.<RowResultAdapter>any());
     verify(serverStream, times(2)).iterator();
     verify(mockStreamingCallable, times(2))
-        .call(Mockito.eq(query), Mockito.any(GrpcCallContext.class));
+        .call(Mockito.any(Query.class), Mockito.any(GrpcCallContext.class));
   }
 
   @Test
-  public void testReadRowsCancel() throws IOException {
-
+  public void testRead100Rows() throws IOException {
```

Comment:
where did testReadRowsCancel() go? we still want the cancel test

Reply:
brought it back